src/katgpucbf/fgpu/pfb.py

```diff
@@ -122,11 +142,24 @@ class PFBFIR(accel.Operation):
         responsibility to zero it when desired, or alternatively to track
         values before and after to measure the change.
 
+    Otherwise, they are
+
+    **in** : samples, complex64
+        Input samples
+    **out** : spectra × channels, complex64
+        See above
+    **weights** : channels*taps, float32
+        See above
+
     Raises
     ------
     ValueError
-        If ``samples`` is not a multiple of 8, or if ``2*channels`` is not a
-        multiple of the workgroup size (currently 128).
+        If ``samples`` is not a multiple of 8 and ``complex_input`` is false
```

Comment:
Or if complex_input is true and input_sample_bits is not 64

Reply:
The ValueError documentation was a bit of a mess (some conditions actually checked on the template were documented here, and a lot of conditions checked on the template were not documented at all). Hopefully my latest push improves things.